Backend/utils/db_utils.py

```diff
@@ -0,0 +1,224 @@
+    return message
+
+def preloadDataFromJson(jsonData, table):
+  if (type(jsonData) == str):
+    items = json.loads(jsonData)
+  else:
+    items = jsonData
+  try:
+    for item in items:
+      if "email" in item.keys() and "password" in item.keys() and "activationId" in item.keys() and "active" in item.keys():
+        email = item['email']
+        password = item['password']
+        activationId = item['activationId']
+        active = item['active']
+
+        table.put_item(
+          Item={
+            'email': email,
+            'password': password,
+            'activationId': activationId,
+            'active': active
+          }
+        )
+      else:
+        message = "Unable to add item to table " + table.describe_table + ". Missing email, password, activationId, or active status"
+        logger.error(message)
+  except IOError as e:
+    message = "Unable to load data into table"
+    logger.exception(message + '\nJSON:\n' + jsonData + '\n' + str(e))
+    return message
+
+def getTable(tableName, dbInstance):
+  try:
+    table = dbInstance.Table(tableName)
+    status = table.table_status
+    return table
+  except ClientError as e:
+    message = "Unable to get table " + tableName + ". Table does not exist"
+    logger.exception(message + '\n' + str(e))
+    return message
+
+def putItem(jsonData, table):
+  if (type(jsonData) == str):
+    item = json.loads(jsonData)
+  else:
+    item = jsonData
+
+  if item is None:
+    message = "Unable to put item. Missing Key"
+    logger.error(message)
+    return message
+
+  response = table.put_item(Item=item)
+  return response
+
+def getItem(jsonData, table):
+  if (type(jsonData) == str):
+    data = json.loads(jsonData)
+  else:
+    data = jsonData
+
+  attributes_to_get = data.get("AttributesToGet")
+  key = data.get("Key")
+
+  if key is None:
+    message = "Unable to get item. Missing Key"
+    logger.error(message)
+    return message
+
+  if attributes_to_get is not None:
+    response = table.get_item(Key=key,AttributesToGet=attributes_to_get)
+  else:
+    response = table.get_item(Key=key)
+  return response
+
+def updateItem(jsonData, table):
+  if (type(jsonData) == str):
+    data = json.loads(jsonData)
+  else:
+    data = jsonData
+
+  key = data.get("Key")
+  update_expression = data.get("UpdateExpression")
+  expression_attribute_values = data.get("ExpressionAttributeValues")
+  return_values = data.get("ReturnValues")
+
+  if key is None:
+    message = "Unable to update item. Missing Key"
+    logger.error(message)
+    return message
+
+  if update_expression is not None and expression_attribute_values is not None and return_values is not None:
+    response = table.update_item(
+      Key=key,
+      UpdateExpression=update_expression,
+      ExpressionAttributeValues=expression_attribute_values,
+      ReturnValues=return_values
+    )
+    return response
+  else:
+    response = table.update_item(Key=key)
+    return response
```

Comment:
I don't know what response is here but I have a feeling it is a dictionary. Same as the above 3. Please fix.